src/components/owned/OwnerInfo.tsx

```diff
@@ -29,82 +28,67 @@ const walletBox = classnames(
 )
 const walletAddress = classnames(display('flex'), flexDirection('flex-col'))
 
-function BadgeTitle({ badge }: { badge: BaseBadgeContract }) {
-  if (badge instanceof EmailBadgeContract) {
-    return (
-      <>
-        <HeaderText extraLeading>
-          This wallet belongs to someone with{' '}
-          <ExternalLink
-            url={getEtherscanAddressUrl(badge.address, Network.Goerli)}
-          >
-            <AccentText bold color="text-secondary">
-              <ContractName
-                hyphens
-                address={badge.address}
-                network={Network.Goerli}
-              />
-            </AccentText>
-          </ExternalLink>
-        </HeaderText>
-      </>
+function replace(
```

Comment:
Maybe let's give it a more explanatory name? Something like `replaceKeywordsInTemplateJSX`?